src/test/resources/pages/instructorFeedbackAllSessionTypes.html

```diff
@@ -758,6 +758,8 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
       <input name="user" type="hidden" value="CFeedbackUiT.instructor">
       <input name="token" type="hidden" value="${sessionToken}">
     </form>
+    <div data-value="" name="editsessiondetails">
```

Comment:
Not necessary; I wasn't aware of that 😅 https://github.com/TEAMMATES/teammates/pull/7615#discussion_r129240543